### PR TITLE
[OSDEV-702] Add Contri Cleaner Serializer basic architecure

### PR DESCRIPTION
**Pull request by killev**
Contri-cleaner basic class and implementation created and implemented:
* class ContriCleanerSerializer that parses and validates data passed to him from any source (json/csv).
* RowCompositeSerializer - class that process data via list of serializers 
  * class RowCleanFieldSerializer - cleans address and name
  * class RowCleanedUserDataSerializer - creates key-value dictionary with cleaned data
  * class RowCountrySerializer - created county id from country name
  * class RowEmptySerializer - process unknown fields that weren't processed by previous processors
  * class RowSectorSerializer - process sector, siplit it into array by ','

Remaining tasks:
* [[OSDEV-893] Implement contri cleaner parser in facility lists processing (facility_list_view_set.py)](https://opensupplyhub.atlassian.net/browse/OSDEV-893)
* [[OSDEV-894] Implement contri cleaner parser in create facility API endpoint (facilities_view_set.py)](https://opensupplyhub.atlassian.net/browse/OSDEV-894)
* [[OSDEV-896] Implement Sector serializer to parse sector, product_type, sector_product_type fields](https://opensupplyhub.atlassian.net/browse/OSDEV-896)
* [[OSDEV-902] Implement required fields serializer](https://opensupplyhub.atlassian.net/browse/OSDEV-902)
* [[OSDEV-903] Add details to doc/release/RELEASE-NOTES.md](https://opensupplyhub.atlassian.net/browse/OSDEV-903)
* [[OSDEV-906] Create a shared interface for Row...Serializer's](https://opensupplyhub.atlassian.net/browse/OSDEV-906)

[OSDEV-702](https://opensupplyhub.atlassian.net/browse/OSDEV-702)

### CODE REVIEW

**Comment by roman-stolar** [comment]
@killev Why you revert ViewSet files?

File: src/django/contricleaner/lib/source_parser_csv.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+class SourceParserCsv(SourceParser):
+    def __init__(self):
+        pass
+
+    def parsed_header(self) -> List[str]:
+        pass
+
+    def parsed_rows(self) -> List[dict]:
+        pass
```
Comment (roman-stolar) [inline src/django/contricleaner/lib/source_parser_csv.py:14]: Why it empty with implementation, is it planed?

**Comment by roman-stolar** [comment]
All other staff looks clean & understandable.

File: src/django/contricleaner/lib/source_parser_csv.py (hunk 1 above)
Comment (roman-stolar) [reply src/django/contricleaner/lib/source_parser_csv.py:14]: Oh I remember, .csv will be only for ListParser and now SourceParser, am I right? @killev

Comment (killev) [reply src/django/contricleaner/lib/source_parser_csv.py:14]: Yes, you're right. I have implementation already

**Review by roman-stolar — DISMISSED** [review]
In general LGTM

**Review by roman-stolar — DISMISSED** [review]

**Comment by killev** [comment]
> @killev Why you revert ViewSet files?

Because I decided to narrow down the scope of Merge request

**Review by roman-stolar — COMMENTED** [review]
Minor comment

File: src/django/contricleaner/lib/serializers/row_serializers/row_clean_field_serializer.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from contricleaner.lib.helpers.clean import clean
+
+
+class RowCleanFieldSerializer:
```
[the remark this answers is not in the document]
Comment (killev) [reply src/django/contricleaner/lib/serializers/row_serializers/row_clean_field_serializer.py:4]: Valid comment.  Created follow-up task https://opensupplyhub.atlassian.net/browse/OSDEV-906

[the remark this answers is not in the document]
Comment (roman-stolar) [reply src/django/contricleaner/lib/serializers/row_serializers/row_clean_field_serializer.py:4]: Okay.

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**